### PR TITLE
18 list of talents skills

### LINKED ISSUE #18: List of talent's skills
- Створити компонент TalentSkill
- Реалізувати відображення навичок таланту відповідно до встановленого стилю через метод масиву map() всередині компонента TalentSkillsList
#8

### PR DESCRIPTION
**Pull request by IhorKopieichykov**
Close #18

### CODE REVIEW

File: src/components/Layout/Header/Header.jsx [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
+				<nav className={styles.nav}>
+					{menuItems.map(({ title, link }, index) => (
+						<a href={link} key={index}>
+							{title}
+						</a>
+					))}
+				</nav>
+				<div className={styles.btns}>
+					<Button className={styles.btn}>Login</Button>
+					<Button className={styles.btn}>Register</Button>
```
Comment (daniilievtukhov) [inline src/components/Layout/Header/Header.jsx:30]: why did u use that kind of syntaxб writing BUTTONS?) better, to write on 1 type all them and for now it's not such useful or IDK)

File: src/components/TalentsListPage/components/SearchPanel/SearchPanel.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Button, Input } from "../../../../shared/components";
+import s from "./SearchPanel.module.scss";
+
+export function SearchPanel() {
+    return (
+        <div className={s.search_panel}>
+            <Input id={s.long} className={s.input} type="text" placeholder="Name"/>
+            <Input className={s.input} type="text" placeholder="Specialization"/>
+            <Input className={s.input} type="text" placeholder="Skill"/>
+            <Button className={s.btn}>Search</Button>
```
Comment (daniilievtukhov) [inline src/components/TalentsListPage/components/SearchPanel/SearchPanel.jsx:10]: why do u use tags with capitals? u just can add classNames for them for facilitation of understanding

File: src/components/Layout/Header/Header.jsx (hunk 1 above)
Comment (IhorKopieichykov) [reply src/components/Layout/Header/Header.jsx:30]: I've been trying to create reusable components

File: src/components/TalentsListPage/components/SearchPanel/SearchPanel.jsx (hunk 2 above)
Comment (IhorKopieichykov) [reply src/components/TalentsListPage/components/SearchPanel/SearchPanel.jsx:10]: this little inconvenience allows us to get rid of code copying

**Review by daniilievtukhov — APPROVED** [review]

File: src/components/TalentsListPage/components/Filters/components/Filter/Filter.module.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    border-bottom: 1px solid #FFFFFF;
+    cursor: pointer;
+    user-select: none;
+    &::after{
+        content: "";
+        position: absolute;
+        right: 10px;
+        width: 16px;
+        height: 10px;
+        background-image: url('./images/arrow.svg');
```
Comment (Ruslanchik01) [inline src/components/TalentsListPage/components/Filters/components/Filter/Filter.module.scss:26]: It is better to use 2 borders than image

File: src/components/TalentsListPage/components/Filters/components/Filter/Filter.module.scss [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    padding: 10px;
+    transform-origin: top center;    
+    transition: all .4s ease-in-out;    
+    &.show{
+        opacity: 1;
+        visibility: visible;
+        max-height: 3000px;
```
Comment (Ruslanchik01) [inline src/components/TalentsListPage/components/Filters/components/Filter/Filter.module.scss]: Why so strange value?

File: src/components/TalentsListPage/components/TalentsList/components/TalentCard/TalentCard.module.scss [hunk 5]
```diff
@@ -0,0 +1,91 @@
+.talent_card {
+	display: block;
+	position: relative;
+	width: 100%;
+	height: 250px;
+	display: flex;
+	flex-direction: row;
+	align-items: flex-start;
+	justify-content: flex-start;
+    gap: 48px;
```
Comment (Ruslanchik01) [inline src/components/TalentsListPage/components/TalentsList/components/TalentCard/TalentCard.module.scss:10]: Recommend you to use same spaces everywhere

**Review by Ruslanchik01 — APPROVED** [review]

**Pull request merged**